qiskit/pulse/instructions/instruction.py

```diff
@@ -267,3 +257,75 @@ def __repr__(self) -> str:
         return "{}({}{})".format(
             self.__class__.__name__, operands, f", name='{self.name}'" if self.name else ""
         )
+
+
+class FrameInstruction(Instruction, ABC):
```

Comment:
Better class name would be `FrameUpdate` or `FrameChange`? (without -Instruction like `Directive` class).  FrameInstruction sounds bit vague to me because it reminds me of something _only_ takes frame, but actual implementation is not. I think our intention for its subclass must be an instruction that updates the frame value, right?

Reply:
Sounds good.